collect_app/src/main/java/org/odk/collect/android/fragments/dialogs/FormsDownloadResultDialog.kt

```diff
@@ -0,0 +1,74 @@
+package org.odk.collect.android.fragments.dialogs
+
+import android.app.Dialog
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AlertDialog
+import androidx.fragment.app.DialogFragment
+import org.jetbrains.annotations.TestOnly
+import org.odk.collect.android.R
+import org.odk.collect.android.formmanagement.downloaderror.FormsDownloadErrorActivity
+import org.odk.collect.android.formmanagement.downloaderror.FormsDownloadErrorItem
+import java.util.ArrayList
+
+class FormsDownloadResultDialog : DialogFragment() {
+    companion object {
+        const val FAILURES = "FAILURES"
```

Comment:
Generally in the Android SDK key names are structure like `<type of key>_<key name>` so this should be `ARG_FAILURES`. Same goes for the other arg key.

Reply:
Fixed.